axelor-human-resource/src/main/resources/i18n/messages_fr.csv

```diff
@@ -961,7 +961,7 @@
 "Register payment","Enregistrer un paiement",,
 "Registration certificate","Certificat d'immatriculation",,
 "Registration code","N° siret",,
-"Registration number","Matricule",,
+"Registration number",,,

```

Comment:
we could do this in another ticket, but we need to manage this, the same key "Registration number" is used for payroll export in HR module, so we must keep this translation. @jmi-axelor maybe we should create a separate ticket to not forget this